tests/Integration/Behaviour/Features/Scenario/Module/module.feature

```diff
@@ -43,6 +43,24 @@ Feature: Module
     Given module ps_featuredproducts has following infos:
       | enabled | true |
 
+  Scenario: Reset module status
+    Given module ps_featuredproducts has following infos:
+      | technical_name | ps_featuredproducts |
+      | enabled        | true |
+      | installed      | true |
+    When I reset module "ps_featuredproducts"
+    Given module ps_featuredproducts has following infos:
+      | technical_name | ps_featuredproducts |
+      | enabled        | true |
+      | installed      | false |
+    Then I should have an exception that module is not installed
+    When I reset module "ps_featuredproducts"
+    Given module ps_featuredproducts has following infos:
+      | technical_name | ps_featuredproducts |
+      | enabled        | false |
+      | installed      | true |
+    Then I should have an exception that module is not enabled
```

Comment:
You can remove the rest for now, in theory you should test that:
- you uninstall the module
- you try to reset
- you get an exception

But since we don't have the uninstall command yet this use case is not testable for the moment, you can leave a comment to remember to add it later when the uninstall command will be available

Reply:
```suggestion
    # Todo: add the check on uninstalled module when the uninstall command is ready
    # Given I uninstall module "ps_featuredproducts"
    # When I reset module "ps_featuredproducts"
    # Then I should have an exception that module is not installed
```